.gitignore

```diff
@@ -20,6 +20,7 @@ latex/
 *.pdf
 *.tag
 .DS_Store
+locm3.sublime-workspace
```

Comment:
That works too, though I believe the file name here is always the same as the project name, and the project file is in the repo.